src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -263,7 +263,7 @@ private void testCreateAndDeleteFeedbackSession() throws InvalidParametersExcept
         } catch (Exception e) {
             assertEquals("The provided feedback session name is not acceptable to TEAMMATES "
                              + "as it cannot contain the following special html characters in brackets: "
-                             + "(&lt; &gt; \\ &#x2f; &#39; &amp;)",
+                             + "(&lt; &gt; &quot &#x2f; &#39; &amp;)",
```

Comment:
This has to be `&quot;` (with the semicolon); the same applies for the one in `FieldValidatorTest.java`